# Paquet unattended-upgrades
# Copyright (C) 2008
# This file is distributed under the same license as the unattended-upgrades package.
# Michael Vogt <mvo@debian.org>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: 0.35\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-07-02 12:27+0200\n"
"PO-Revision-Date: 2015-07-02 12:30+0200\n"
"Last-Translator: Alexandre Detiste <alexandre@detiste.be>\n"
"Language-Team: French <debian-l10n-french@lists.debian.org>\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../unattended-upgrade:189
#, python-format
msgid "Progress: %s %% (%s)"
msgstr "Progression : %s %% (%s)"

#: ../unattended-upgrade:383 ../unattended-upgrade:400
#, python-format
msgid "Unable to parse %s."
msgstr ""

#: ../unattended-upgrade:486 ../unattended-upgrade:608
#: ../unattended-upgrade-shutdown:193
msgid "All upgrades installed"
msgstr "Toutes les mises à niveau ont été installées"

#: ../unattended-upgrade:488 ../unattended-upgrade:600
msgid "Installing the upgrades failed!"
msgstr "Échec de l'installation des mises à niveau"

#: ../unattended-upgrade:489 ../unattended-upgrade:601
#: ../unattended-upgrade:755 ../unattended-upgrade:757
#: ../unattended-upgrade:1432 ../unattended-upgrade:1434
#: ../unattended-upgrade-shutdown:140
#, python-format
msgid "error message: '%s'"
msgstr "Message d'erreur : %s"

#: ../unattended-upgrade:490 ../unattended-upgrade:602
#, python-format
msgid "dpkg returned a error! See '%s' for details"
msgstr "dpkg a renvoyé une erreur. Voir « %s » pour les détails"

#: ../unattended-upgrade:572
#, fuzzy, python-format
#| msgid "Packages that will be upgraded: %s"
msgid "Some packages can not be upgraded: %s"
msgstr "Paquets mis à niveau: %s"

#: ../unattended-upgrade:754 ../unattended-upgrade:756
#: ../unattended-upgrade:1431 ../unattended-upgrade:1433
#: ../unattended-upgrade-shutdown:139
#, fuzzy
#| msgid "dpkg returned an error! See '%s' for details"
msgid "Apt returned an error, exiting"
msgstr "dpkg a renvoyé une erreur! Voir '%s' pour les détails"

#: ../unattended-upgrade:942
msgid ""
"No '/usr/bin/mail' or '/usr/sbin/sendmail',can not send mail. You probably "
"want to install the 'mailx' package."
msgstr ""
"L'exécutable '/usr/bin/mail' ou '/usr/sbin/sendmail' est introuvable,  "
"impossible d'envoyer un courrier électronique. Veuillez installer le paquet "
"« mailx »."

#: ../unattended-upgrade:956
msgid "[reboot required]"
msgstr ""

#: ../unattended-upgrade:958
msgid "[package on hold]"
msgstr ""

#: ../unattended-upgrade:961
#, fuzzy, python-brace-format
#| msgid "[reboot required] unattended-upgrades result for '%s': %s"
msgid ""
"{hold_flag}{reboot_flag} unattended-upgrades result for '{machine}': {result}"
msgstr "[Redémarrage requis] résultat de unattended-upgrades pour '%s': %s"

#: ../unattended-upgrade:965
#, python-format
msgid ""
"Unattended upgrade returned: %s\n"
"\n"
msgstr ""
"La mise à niveau automatique a renvoyé : %s\n"
"\n"

#: ../unattended-upgrade:968
msgid ""
"Warning: A reboot is required to complete this upgrade.\n"
"\n"
msgstr ""
"Avertissement : un redémarrage est requis afin de compléter cette mise à "
"niveau.\n"

#: ../unattended-upgrade:970
msgid "Packages that were upgraded:\n"
msgstr "Paquets mis à niveau:\n"

#: ../unattended-upgrade:972
msgid "Packages that attempted to upgrade:\n"
msgstr "Essai de mise à niveau des paquets:\n"

#: ../unattended-upgrade:976
msgid "Packages with upgradable origin but kept back:\n"
msgstr ""
"Paquets susceptibles d'une mise à niveau mais maintenus dans leur état "
"actuel :\n"

#: ../unattended-upgrade:981
msgid "Package installation log:"
msgstr "Journal d'installation des paquets:"

#: ../unattended-upgrade:984
msgid "Unattended-upgrades log:\n"
msgstr "Journal de unattended-upgrades:\n"

#: ../unattended-upgrade:1012
#, python-format
msgid "Writing dpkg log to '%s'"
msgstr "Écriture du journal de dpkg dans « %s »"

#: ../unattended-upgrade:1168
#, python-format
msgid "Found %s, but not rebooting because %s is logged in."
msgid_plural "Found %s, but not rebooting because %s are logged in."
msgstr[0] "Trouvé %s, mais ne redémarre pas car %s est connecté."
msgstr[1] "Trouvé %s, mais ne redémarre pas car %s sont connectés."

#: ../unattended-upgrade:1222
#, python-format
msgid "package '%s' upgradable but fails to be marked for upgrade (%s)"
msgstr ""
"Le paquet « %s » peut être mis à niveau mais échoue à être marqué comme tel "
"(%s)"

#: ../unattended-upgrade:1297
#, python-format
msgid "Packages that are auto removed: '%s'"
msgstr "Suppression automatique des paquets: %s"

#: ../unattended-upgrade:1303
msgid "Packages were successfully auto-removed"
msgstr "Ces packets ont été correctement supprimés"

#: ../unattended-upgrade:1305
msgid "Auto-removing the packages failed!"
msgstr "Auto-remove de ces paquets a échoué!"

#: ../unattended-upgrade:1306
#, python-format
msgid "Error message: '%s'"
msgstr "Message d'erreur: %s"

#: ../unattended-upgrade:1307
#, python-format
msgid "dpkg returned an error! See '%s' for details"
msgstr "dpkg a renvoyé une erreur! Voir '%s' pour les détails"

#: ../unattended-upgrade:1367
#, python-format
msgid "Initial blacklisted packages: %s"
msgstr "Paquets initialement sur la liste noire: %s"

#: ../unattended-upgrade:1372
#, python-format
msgid "Initial whitelisted packages: %s"
msgstr "Paquets initialement sur la liste blanche: %s"

#: ../unattended-upgrade:1375
msgid "Starting unattended upgrades script"
msgstr "Démarrage du script de mise à niveau automatique"

#: ../unattended-upgrade:1384
#, python-format
msgid "Allowed origins are: %s"
msgstr "Les origines permises sont: %s"

#: ../unattended-upgrade:1400 ../unattended-upgrade:1401
msgid "Unclean dpkg state detected, trying to correct"
msgstr "détection d'un état de dpkg impropre, tentative de correction"

#: ../unattended-upgrade:1412
#, python-format
msgid ""
"dpkg --configure -a output:\n"
"%s"
msgstr ""
"sortie de « dpkg --configure -a »:\n"
"%s"

#: ../unattended-upgrade:1421
msgid "Lock could not be acquired (another package manager running?)"
msgstr ""
"Verrouillage impossible (y a-t-il un autre gestionnaire de paquets en cours "
"d'exécution ?)"

#: ../unattended-upgrade:1423
msgid "Cache lock can not be acquired, exiting"
msgstr "Le verrouillage du cache est impossible, abandon"

#: ../unattended-upgrade:1438 ../unattended-upgrade:1439
#: ../unattended-upgrade:1633 ../unattended-upgrade:1634
msgid "Cache has broken packages, exiting"
msgstr "Le cache contient des paquets cassés, abandon"

#: ../unattended-upgrade:1488
#, python-format
msgid "GetArchives() failed: '%s'"
msgstr "Échec de GetArchives(): %s"

#: ../unattended-upgrade:1504 ../unattended-upgrade:1505
#, python-format
msgid "An error occurred: '%s'"
msgstr "Une erreur s'est produite: %s"

#: ../unattended-upgrade:1507 ../unattended-upgrade:1509
#, python-format
msgid "The URI '%s' failed to download, aborting"
msgstr "Échec du téléchargement à l'URI « %s », abandon"

#: ../unattended-upgrade:1513
#, python-format
msgid "Download finished, but file '%s' not there?!?"
msgstr "Téléchargement terminé mais le fichier « %s » est absent !"

#: ../unattended-upgrade:1529 ../unattended-upgrade:1533
#, python-format
msgid "Package '%s' has conffile prompt and needs to be upgraded manually"
msgstr ""
"Le paquet « %s » provoque un conflit de fichiers de configuration et "
"nécessite une mise à niveau manuelle"

#: ../unattended-upgrade:1561
#, python-format
msgid "package '%s' not upgraded"
msgstr "Le paquet « %s » n'a pas été mis à niveau"

#: ../unattended-upgrade:1580
msgid ""
"No packages found that can be upgraded unattended and no pending auto-"
"removals"
msgstr ""
"Aucun paquet à mettre à niveau automatiquement ni à supprimer automatiquement"

#: ../unattended-upgrade:1610
#, python-format
msgid "Packages that will be upgraded: %s"
msgstr "Paquets mis à niveau: %s"

#: ../unattended-upgrade:1713
msgid "print debug messages"
msgstr "Afficher les messages de débogage"

#: ../unattended-upgrade:1716
msgid "make apt/libapt print verbose debug messages"
msgstr "Afficher les messages de débogage de apt/libapt"

#: ../unattended-upgrade:1719
msgid "print info messages"
msgstr "Afficher les messages d'information"

#: ../unattended-upgrade:1722
msgid "Simulation, download but do not install"
msgstr "Simulation, télécharge mais n'installe pas"

#: ../unattended-upgrade:1725
#, fuzzy
#| msgid "Simulation, download but do not install"
msgid "Only download, do not even try to install."
msgstr "Simulation, télécharge mais n'installe pas"

#: ../unattended-upgrade:1728
msgid "Upgrade in minimal steps (and allow interrupting with SIGTERM"
msgstr ""
"Mise à niveau par étapes minimales (et autoriser l'interruption avec SIGTERM)"

#: ../unattended-upgrade:1737
msgid "You need to be root to run this application"
msgstr ""
"Vous devez posséder les privilèges du superutilisateur pour exécuter cette "
"application"

#: ../unattended-upgrade-shutdown:76
#, fuzzy
#| msgid "Unattended-upgrade in progress during shutdown, sleeping for 5s"
msgid ""
"Unattended-upgrade in progress during shutdown, please don't turn off the "
"computer"
msgstr "Unattended-upgrade en cours durant l'extinction, en pause pour 5 s"

#: ../unattended-upgrade-shutdown:150
msgid "Running unattended-upgrades in shutdown mode"
msgstr "Exécuter unattended-upgrades lors de l'extinction"

#: ../unattended-upgrade-shutdown:176
#, python-format
msgid "Giving up on lockfile after %s delay"
msgstr "Renonciation sur un fichier verrou après un délai de %s."

#: ../unattended-upgrade-shutdown:189
msgid ""
"Unattended-upgrades stopped. There may be upgrades left to be installed in "
"the next run."
msgstr ""

#~ msgid "unattended-upgrades result for '%s': '%s'"
#~ msgstr "résultat de unattended-upgrades pour '%s' : '%s'"

#~ msgid "Progress: %s %%: (%s)"
#~ msgstr "Progression : %s %% : (%s)"
